src/Compilers/CSharp/csc/Csc.cs

```diff
@@ -11,6 +11,8 @@ namespace Microsoft.CodeAnalysis.CSharp.CommandLine
 {
     internal sealed class Csc : CSharpCompiler
     {
+        private SimpleAnalyzerAssemblyLoader _analyzerAssemblyLoader;
```

Comment:
Please make this the interface and pass the value to the constructor.  The concrete type uses APIs which are not valid in CoreFx and hence this breaks the CoreFx build.